src/main/java/frc/robot/commands/ShooterPrime.java

```diff
@@ -23,30 +23,31 @@ public ShooterPrime(ShooterSubsystem shooterSubsystem) {
   @Override
   public void initialize() {
     m_ShooterSubsystem.prime();
-
+    i = 0;
   }
 
   // Called every time the scheduler runs while the command is scheduled.
   @Override
   public void execute() {
-    
+
   }
 
   // Called once the command ends or is interrupted.
   @Override
   public void end(boolean interrupted) {
-    if (!interrupted){
-      //TODO: uncomment when cargoMotor exists 
-      //m_ShooterSubsystem.shoot()
-    }
-    m_ShooterSubsystem.windFlywheel(0); //TODO: Look at this again, we don't want our flywheel to instantly switch off in the end
+    m_ShooterSubsystem.runCargo(false);
+    m_ShooterSubsystem.windFlywheel(0);
   }
 
   // Returns true when the command should end.
   @Override
   public boolean isFinished() {
     if(m_ShooterSubsystem.wheelReady()){
-      return true;
+      m_ShooterSubsystem.runCargo(true);
+      i++;
+      if(i==10){
```

Comment:
should put a quick comment here to remind us that it is expected to run for 10*20ms